lib/workers/repository/update/pr/participants.ts

```diff
@@ -7,10 +7,19 @@ import { sampleSize } from '../../../../util/sample';
 import { codeOwnersForPr } from './code-owners';
 
 async function addCodeOwners(
+  config: RenovateConfig,
   assigneesOrReviewers: string[],
   pr: Pr
 ): Promise<string[]> {
-  return [...new Set(assigneesOrReviewers.concat(await codeOwnersForPr(pr)))];
+  const codeOwners = await codeOwnersForPr(pr);
+  const normalizedCodeOwners = [...new Set(codeOwners.map(noLeadingAtSymbol))];
```

Comment:
why remove the `@` symbol? shouldn't it be used to detect groups instead of doing failing API requests?

Reply:
Using the @ symbol is not a fool-proof way to differentiate users from groups, since while user emails don't require the beginning @, usernames do. Still, I can update the PR to filter the user emails from the list, but there will still be failing API requests where something like `@username` is used.

Additionally, removing the @ symbol is required when searching for a Group against the Gitlab API, so it needs to happen before the request is made.

Reply:
Let me know if I should proceed w/ filtering out emails from the list. I think its fine to have the list normalized here since it needs to be normalized shortly after in the same flow.

Reply:
ok, at least filter emails so we can reduce failed. requests. I also think we should remove the `@` directly before the request

Reply:
removing the `@` here can have negative impact on other platform.

so please do the email filtering on gitlab platform code.